readthedocs/core/signals.py

```diff
@@ -98,16 +98,7 @@ def delete_projects_and_organizations(sender, instance, *args, **kwargs):
                                           ).exclude(num_users__gt=1)
     )
 
-    # Here we count the users list from the organization that the user belong
-    # Then exclude the organizations where there are more than one user
-    oauth_organizations = (
-        RemoteOrganization.objects.annotate(num_users=Count('users')
-                                            ).filter(users=instance.id
-                                                     ).exclude(num_users__gt=1)
-    )
```

Comment:
Why did we remove this code? Do we run it somewhere else?

Reply:
We are not deleting `RemoteOrganization` on user delete as we will only delete the `RemoteOrganizationRelation` in this case and that is handled by model `on_delete=CASECADE`.

Reply:
If we want, we could do a cleanup of organizations/repositories at some point by deleting those that has 0 users connected. I don't think we need this _now_, tho.